plugins/api-docs/src/components/ApiExplorerPage/DefaultApiExplorerPage.tsx

```diff
@@ -37,11 +37,38 @@ import {
   CatalogFilterLayout,
   EntityOwnerPickerProps,
 } from '@backstage/plugin-catalog-react';
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { registerComponentRouteRef } from '../../routes';
 import { usePermission } from '@backstage/plugin-permission-react';
 import { catalogEntityCreatePermission } from '@backstage/plugin-catalog-common/alpha';
 
+/**
+ * Props for default filters.
+ *
+ * @public
+ */
+export type DefaultFiltersProps = {
+  initialKind?: string;
+  initiallySelectedFilter?: UserListFilterKind;
+  ownerPickerMode?: EntityOwnerPickerProps['mode'];
+  initiallySelectedNamespaces?: string[];
+};
+
+/** @public */
+export const DefaultFilters = (props: DefaultFiltersProps) => {
+  const { initiallySelectedFilter, ownerPickerMode } = props;
```

Comment:
This look pretty similar to the `DefaultFilters` provided by the `catalog-react` package.
https://github.com/backstage/backstage/blob/e4cfef8d87ddeccd0668f17731c111605b62dc76/plugins/catalog-react/src/components/DefaultFilters/DefaultFilters.tsx#L44
I wonder if it would be a good idea to make that component more generic so that it can be used in `TechDocs` and `ApiDocs` plugins as well?

Reply:
It might be nice also be able to disable some of the default filters through the configuration file (no code required). What do you think @backstage/maintainers?